### DISCUSSION TITLE
Building multi-buffer on top of org-transclusion?

### DISCUSSION THREAD

**Discussion by joshcho** [nobiot/org-transclusion#200] [Ideas]
Can we use org-transclusion to build something like [multi-buffer in Zed](https://youtu.be/wXT73bBr83s?t=1132)? Feels quite useful.

**Comment by nobiot** [comment]
Back from vacation :)

I feel that Emacs users might be better served by a combination of `grep (ripgrep)` and `wgrep`.

Below is a silly example of finding all the occurrences of "transclusion-" in the `org-transclusion` project and replacing them  with "transclusion-hello" across all the files. I have un-done this since.

![Screenshot from 2023-08-24 09-20-47](https://github.com/nobiot/org-transclusion/assets/12507865/29575d24-4e11-45ff-9d9e-4fb2d56816d0)

![Screenshot from 2023-08-24 09-20-58](https://github.com/nobiot/org-transclusion/assets/12507865/12dbb924-7523-4cc4-a048-c261fbba5289)

I am no developer by trade so I may be missing the point of multi-buffer capability in Zed (or multi-buffer in general if this is a general concept).
